CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+
+## 1.0.0
+
+#### Breaking Changes
+
+If you used deepstream 0.x before you may need to change some API calls.
+
+###### Permission Handler
+
+In 0.x you can set a `permissionHandler` which needs to implement two functions:
+
+- `isValidUser(connectionData, authData, callback)`
+- `canPerformAction(username, message, callback)`
+
+In deepstream 1.0 the `canPerformAction` is no longer part of the `permissionHandler` instead
+it is moved to a new handler: `authenticationHandler`.
+
+###### Plugin API
+
+All connectors, the `permissionHandler`, `authenticationHandler` and the Logger
+are implenting deepstream's plugin API which needss to export a constructor with
+these feautres:
+
+- either provide a `isReady` property which is true after calling the constructor. This makes only sense for a synchronous initialization.
+- otherwise (`isReady` is false) it needs to emit an `ready` event
+- for errors it should emit an `error` event
+
+###### Logger and colors options
+
+The color flag can't be set in the root level of the configuration anymore.
+The default logger will print logs to the StdOut/StdErr in colors.
+You can use the [deepstream.io-logger-winston](https://www.npmjs.com/package/deepstream.io-logger-winston) which can be configured in the config.yml file with several options.
+
+
```

Comment:
I think how the configuration options are passed and set is different as well? Before it was `ds.set` now you pass it in the constructor.